utils/isTrackingEnabled.ts

```diff
@@ -0,0 +1,6 @@
+import { getEnvVariables } from '@prezly/theme-kit-nextjs';
+import type { GetServerSidePropsContext } from 'next';
+
+export function isTrackingEnabled(context: GetServerSidePropsContext): boolean {
+    return getEnvVariables(context.req).PREZLY_MODE !== 'preview';
```

Comment:
So the reason for this approach is that even though we've added the variable to the `X-Prezly-Env` header, we didn't actually extract and use it (I thought it was somehow automagic) 😄